bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -353,6 +357,7 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
     serialize(outStream, dbCheckPointWindowSize);
     serialize(outStream, dbCheckpointDirPath);
     serialize(outStream, dbSnapshotIntervalSeconds);
+    serialize(outStream, dbCheckpointMonitorIntervalSeconds);
```

Comment:
Can we also add dbSnapshotIntervalSeconds && dbCheckpointMonitorIntervalSeconds in the print where we print all replica configs in the log

Reply:
done.